rootfs-builder/clearlinux/Dockerfile.am

```diff
@@ -0,0 +1,6 @@
+From fedora:27
```

Comment:
This line is a bit of a problem. If we leave it like this, it'll probably break at some future date.

We *could* add one extra variable to each `config.sh` that specifies the version for that particular distro. For example, `fedora/config.sh` would become something like:
```
FEDORA_OS_VERSION=27
OS_VERSION=${OS_VERSION:-${FEDORA_OS_VERSION}}
```
But we'd then need to do some rework to allow the clearlinux rootfs build to use `$FEDORA_OS_VERSION` so that the clearlinux `Dockerfile` could specify:

```
From @FEDORA_OS_VERSION@
```

But this all seems rather odd. Clear Linux is the only distro that currently requires a *different* distro to build its rootfs. So why can't we just build a clearlinux rootfs using... clearlinux:

```
From clearlinux:@OS_VERSION@
```

Where `@OS_VERSION@` would probably be just "`latest`"?

Reply:
Issue raised as #48.